bindings/go/src/fdb/fdb_test.go

```diff
@@ -313,3 +313,24 @@ func ExamplePrintable() {
 	fmt.Println(fdb.Printable([]byte{0, 1, 2, 'a', 'b', 'c', '1', '2', '3', '!', '?', 255}))
 	// Output: \x00\x01\x02abc123!?\xff
 }
+
+func ExampleOpenWithConnectionString() {
+	var e error
+
+	e = fdb.APIVersion(API_VERSION)
+	if e != nil {
+		fmt.Printf("Unable to set API version: %v\n", e)
+		return
+	}
+
+	// OpenWithConnectionString opens the database described by the connection string
+	db, e := fdb.OpenWithConnectionString("")
```

Comment:
I have to see if there is a good way to get the cluster file here.

Reply:
Probably implementing the logic from `ClusterConnectionFile::lookupClusterFileName` is the easiest approach here.